skbio/tree/_tree.py

```diff
@@ -583,36 +591,45 @@ def lowest_common_ancestor(self, tipnames):
         root
 
         """
-        nodes = [self.find(x) for x in tipnames]
+        if kwargs and "tipnames" in kwargs:
+            nodes = kwargs["tipnames"]
```

Comment:
Does this design assume that `nodes` and `tipnames` would never be passed at the same time? If the two parameters are passed at the same time, `tipnames` will override `nodes`, which may not be desired behavior. Probably better to ensure only one or the other is passed. However, I don't think this is a likely scenario, so it may not be necessary to address.

Reply:
This means if `tipnames` is passed, it will override `nodes`. This design isn't perfect. But I am not aware of an elegant way to alias a parameter of a Python function. To avoid the unwanted scenario you pointed out, I think we can do this:

```python
if nodes is None and kwargs and "tipnames" in kwargs:
```